### PR TITLE
Feature/UI component buildout

### LINKED ISSUE #16: CSS: Main Page - Form
See miroboard:

https://miro.com/app/board/o9J_l7sDZXE=/

### LINKED ISSUE #33: CSS/JSX: Nav Menu (Burger Time)
Build out hamburger menu display/functionality using react-burger-menu npm package:
https://www.npmjs.com/package/react-burger-menu

### PR DESCRIPTION
**Pull request by pcmueller**
Type of Change
- [x] JSX
- [ ] HTML
- [x] JavaScript
- [x] CSS
- [x] React
- [ ] React Router
- [ ] Cypress
- [ ] Apollo
- [ ] GraphQL
- [ ] Twilio API
- [ ]Refactor/Fix Bugs

## Please describe the change made
UI features

## Please give a short description of the changes that were made below
- Feature:
- Built out hamburger dropdown menu JSX and CSS
- Built array method to convert fetched Contacts API data into required dropdown menu (react-select) format (_now displaying_)
- Converted "modes of transportation" input into dropdown menu
- Built  `travelModeData` dataset and connected to dropdown menu (_now displaying_)
- Refactored all main page CSS and child components' layout and styling

## How has this been tested?
- [x] In Browser
- [ ] In Console
- [ ] Cypress
- [ ] Postman

Screen Shot 2021-07-12 at 8 23 09 PM

### CODE REVIEW

File: walk-safe/src/Components/Form/Form.js [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
+import transportOptions from '../../assets/travelModeData';
 import TripETA from '../TripETA/TripETA';
 // import TripDuration from '../TripDuration/TripDuration';
 
 function Form({contacts}) {
 
-  const [etaModalIsOpen, setEtaModalIsOpen] = useState(true);
-  const [selectedContact, setSelectedContact] = useState([]);
+  const [etaModalIsOpen, setEtaModalIsOpen] = useState(false);
+  const [formattedContacts, setFormattedContacts] = useState([]);
```
Comment (bhohnco) [inline walk-safe/src/Components/Form/Form.js:10]: Great job with setting up these hooks. Looks clean and readable 👀

File: walk-safe/src/Components/Form/Form.js [hunk 2]
```diff
@@ -35,24 +48,20 @@ function Form({contacts}) {
...
+        defaultValue={selectedTransport}
+        onChange={setSelectedTransport}
+        options={transportOptions}
       />
       <Select
         className='dropdown'
         placeholder='Select contact'
+        value={selectedContact}
         defaultValue={selectedContact}
         onChange={setSelectedContact}
```
Comment (bhohnco) [inline walk-safe/src/Components/Form/Form.js:63]: Great add here

File: walk-safe/src/Components/NavBar/_navbar.scss [hunk 3]
```diff
@@ -2,20 +2,99 @@
...
+  width: 40px;
+}
+
+/* Color/shape of close button cross */
+.bm-cross {
+  background: #bdc3c7;
+}
+
+/*
+Sidebar wrapper styles
```
Comment (bhohnco) [inline walk-safe/src/Components/NavBar/_navbar.scss:70]: Looking forward to pulling this down and messing with it!

Comment (pcmueller) [reply walk-safe/src/Components/NavBar/_navbar.scss:70]: Please do!

**Review by carolineshowscode — APPROVED** [review]
This looks awesome. Seems like our next step will to be making the Form component fields for address have an autocomplete address capability (google maps)

**Review by bhohnco — APPROVED** [review]

**Pull request merged**